A36-xds-for-servers.md

```diff
@@ -95,23 +136,27 @@ The `GRPC_XDS_BOOTSTRAP` file will be enhanced to have a new field:
 
 XdsServer will use the normal `XdsClient` to communicate with the xDS server.
 There is no default value for `server_listener_resource_name_template` so if it
-is not present in the bootstrap then server creation or start will fail.
-The XdsServer will pass the listening address (commonly using a wildcard IP
-address, like `::` or `0.0.0.0`) to a new XdsClient API to start a watch for a
-`envoy.config.listener.v3.Listener` resource. XdsClient will perform the `%s`
-replacement if the token is present and watch the corresponding listener
-resource. No special character handling of the template or its replacement is
-performed. For example, with an address of `[::]:80` and a template of
-`grpc/server?xds.resource.listening_address=%s`, the resource name would be
-`grpc/server?xds.resource.listening_address=[::]:80`.
-
-The xDS-returned Listener must have an [`address`][Listener.address] that
-matches the listening address provided. The Listener's `address` would be a TCP
-`SocketAddress` with matching `address` and `port_value`. The XdsClient must
-NACK the resource if the address does not match. The xDS client must also NACK
-the resource if `Listener.listener_filters` is non-empty.
-
-Although `FilterChain.filters`s will not be observed initially, the
+is not present in the bootstrap then server creation or start will fail or the
+XdsServer will become "not serving". The XdsServer will pass the listening
+address (commonly using a wildcard IP address, like `::` or `0.0.0.0`) to a new
```

Comment:
I think the template expansion will actually be done by the XdsServer code, not by the XdsClient.  The XdsServer will expand the template and then use the expanded name to start a Listener watch on the XdsClient.  The Listener watch will by done via the existing XdsClient API for Listener watches (i.e., the same one used in the gRPC client).